src/View/Helper/PaginatorHelper.php

```diff
@@ -1121,4 +1121,31 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Dropdown array for Select limit
+     *
+     * @param array $limits This is array of options.
+     * @param array $default Default limit for option selecting. Default is Paginator limit.
+     *
+     * @return string html output.
+     */
+    public function showLimit($limits = [], $default = 0)
```

Comment:
Why "show"? Maybe just `limit()` (like "numbers" and "last"/"prev") would also work?
Let's wait for more feedback here.